desktop_server/src/com/mygdx/game/server/network/Server.java

```diff
@@ -36,41 +36,22 @@ public Server(
   }
 
   private void handle(int client, WebSocketFrame frame) {
-    Object[] commands = Arrays.stream(frame.textData().split(":")).toArray();
-    String type = (String) commands[0];
+    var commands = frame.textData().split(":");
+    var type = commands[0];
+    log.info("Received frame: " + frame.textData() + " from " + client);
     switch (type) {
-      case "map" -> {
-        var map = mapInitializer.initializeMap();
-        clientManager.getClients().values()
-          .forEach(webSocket -> {
-            var array = map.entrySet().stream()
-                .map(entry -> new Object[] {entry.getKey(), entry.getValue()})
-                .toArray();
-            var header = Arrays.stream(new Object[] {Arrays.stream(new Object[] {"map", "map"}).toArray()}).toArray();
-            var mapArray = Stream.concat(Arrays.stream(header), Arrays.stream(array)).toArray();
-            webSocket.write(Json.encodeToBuffer(mapArray));
-          });
-      }
-      case "unit" -> {
-        var map = unitInitializer.initializeTestUnit(0);
-        clientManager.getClients().values()
-          .forEach(webSocket -> {
-            var array = map.entrySet().stream()
-                    .map(entry -> new Object[] {entry.getKey(), entry.getValue()})
-                    .toArray();
-            var header = Arrays.stream(new Object[] {Arrays.stream(new Object[] {"unit", "unit"}).toArray()}).toArray();
-            var mapArray = Stream.concat(Arrays.stream(header), Arrays.stream(array)).toArray();
-            webSocket.write(Json.encodeToBuffer(mapArray));
-          });
-      }
+      case "map" -> mapInitializer.initializeMap(client);
```

Comment:
te 2 linijki ok, nie trzeba już takiej mapy

ale trzeba zapełnić po tym wszystkim GameState u klienta, jak zobaczyłem, że jest pusty to wtedy zacząłem zmieniać te koordynaty, ale źle zrobiłem